### PR TITLE
Improve automobile documentation

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Address  #5609: improve automobile documentation and vehicle PROTOs:
* [x] fix default translation of "RangeRoverSportSVRSimple" and "TruckSimple" not updated to ENU.
* [x] improve description in vehicles PROTO files (fix spelling, add missing sentence stop, etc.)
* [x] add field description in vehicles PROTO to automatically generate the documentation 
* [x] add a page for each vehicle group (bmw, citroen, generic, etc.)
* [x] fix a couple of issues in `icon_creator`: passing an integer to the python `Field.setSFFloat` is now throwing an error
* [x] reuse `generate_object_doc.py` for generating the vehicles documentation

Documentation: https://www.cyberbotics.com/doc/automobile/index?version=documentation-add-tractor

### CODE REVIEW

File: docs/automobile/vehicle-generic.md [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+  SFString   window            "automobile"
+}
+```
+
+> **File location**: "[WEBOTS\_HOME/projects/vehicles/protos/generic/Truck.proto]({{ url.github_tree }}/projects/vehicles/protos/generic/Truck.proto)"
+
+> **License**: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+[More information.](https://cyberbotics.com/webots_assets_license)
+
+### Truck Field Summary
```
Comment (ygoumaz) [inline docs/automobile/vehicle-generic.md:393]: Why is there a field summary only for the Truck and for the TruckTrailer? It doesn't seem very consistent. The Bus doesn't have a summary for example.

**Review by ygoumaz — CHANGES_REQUESTED** [review]
Everything seems fine. Thank you! I just have a question about consistency in `vehicle-generic.md`.

File: docs/automobile/vehicle-generic.md (hunk 1 above)
Comment (stefaniapedrazzi) [reply docs/automobile/vehicle-generic.md:393]: You are right.
All the `vehicles-*.md` files are automatically generated by the script, but it seems indeed wrong that for example the Bus.color field is not documented.
I will check the documentation script.

Comment (stefaniapedrazzi) [reply docs/automobile/vehicle-generic.md:393]: I'm adding the comments in the PROTO files so that the documentation is generated correctly.

**Review by ygoumaz — APPROVED** [review]
Thank you! It is much more consistent now.

**Pull request merged**